pages.fr/common/git-clone.md

```diff
@@ -1,24 +1,36 @@
 # git clone
 
-> Clone un dépôt existant.
+> Cloner un dépôt existant.
 > Plus d'informations : <https://git-scm.com/docs/git-clone>.
 
-- Clone un dépôt existant :
+- Cloner un dépôt existant :
 
-`git clone {{location_du_depot_distant}}`
+`git clone {{emplacement_du_depot_distant}}`
 
-- Clone un dépôt existant et ses sous-modules :
+- Cloner un dépôt existant dans un répertoire spécifique:
 
-`git clone --recursive {{location_du_depot_distant}}`
+`git clone {{emplacement_du_depot_distant}} {{chemin/vers/repertoire}}`
 
-- Clone un dépôt local :
+- Cloner un dépôt existant et ses sous-modules :
 
-`git clone -l`
+`git clone --recursive {{emplacement_du_depot_distant}}`
 
-- Clone silencieusement :
+- Cloner un dépôt local :
 
-`git clone -q`
+`git clone -l {{chemin/vers/depot/local}}`
 
-- Clone un dépôt existant en ne récupérant que les 10 commits les plus récents sur la branche par défaut (plus rapide) :
+- Cloner silencieusement :
 
-`git clone --depth {{10}} {{location_du_depot_distant}}`
+`git clone -q {{emplacement_du_depot_distant}}`
+
+- Cloner un dépôt existant en ne récupérant que les 10 commits les plus récents sur la branche par défaut (plus rapide):
+
+`git clone --depth {{10}} {{emplacement_du_depot_distant}}`
+
+- Cloner un dépôt existant en ne récupérant qu'une branche spécifique:
```

Comment:
```suggestion
- Cloner un dépôt existant en ne récupérant qu'une branche spécifique :
```
:innocent:

Reply:
Ok, thanks
But, [`CONTRIBUTING.md`](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md) and [`style-guide.md`](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) say:
> **Markdown format**
>As a quick reference, the format of each page should match the following template:
```
# command-name

> Short, snappy description.
> Preferably one line; two are acceptable if necessary.
> More information: <https://url-to-upstream.tld>.

- Example description:

`command -opt1 -opt2`

- Example description:

`command -opt1 -opt2 -arg1 {{arg_value}}`
```
Without the space before the colons.  
I checked in `pages`, `pages.es`, `.de`, and `.nl`  and they don't put the space. Strangely, all french pages do 😄

Reply:
@Stijn-Bch It's a French grammar rule.